ouroboros-network/sim-tests-lib/Test/Ouroboros/Network/PeerSelection/Cardano/PublicRootPeers.hs

```diff
@@ -17,7 +20,8 @@ import Test.Ouroboros.Network.PeerSelection.Instances
 import Test.Ouroboros.Network.Utils (ShrinkCarefully, prop_shrink_nonequal,
            prop_shrink_valid)
 
-
+import Ouroboros.Cardano.Network.PublicRootPeers (CardanoExtraPeers (..))
+import Ouroboros.Cardano.Network.PublicRootPeers qualified as CNPRP
```

Comment:
Better qualification, e.g `PublicRoots`.  Any other places where it's luring? Please grep for `CNPRP` etc.